GIT-INFO.md

```diff
@@ -27,16 +27,12 @@ In environments that don't support configure (i.e. Windows), do this:
 
 ## REQUIREMENTS
 
-For `autoreconf` and `configure` (not `buildconf.bat`) to work, you need the
-following software installed:
-
- o autoconf 2.57  (or later)
- o automake 1.7   (or later)
- o libtool  1.4.2 (or later)
- o GNU m4 (required by autoconf)
- o perl
+See [docs/INTERNALS.md][0] for information on the requirements that you'll
+need or `autoreconf` and `configure` (not `buildconf.bat`) to work.
```

Comment:
```suggestion
See [docs/INTERNALS.md][0] for requirement details.
```

... and I think the next paragraph can be completely removed.

Reply:
Thanks! How does this look now?